mobile/lib/services/album.service.dart

```diff
@@ -62,60 +54,14 @@ class AlbumService {
     final Stopwatch sw = Stopwatch()..start();
     bool changes = false;
     try {
-      final List<String> excludedIds =
-          await _backupService.excludedAlbumsQuery().idProperty().findAll();
-      final List<String> selectedIds =
-          await _backupService.selectedAlbumsQuery().idProperty().findAll();
-      if (selectedIds.isEmpty) {
-        final numLocal = await _db.albums.where().localIdIsNotNull().count();
-        if (numLocal > 0) {
-          _syncService.removeAllLocalAlbumsAndAssets();
-        }
-        return false;
-      }
       final List<AssetPathEntity> onDevice =
           await PhotoManager.getAssetPathList(
```

Comment:
Maybe we can just get the `Recents` album here, which has all the assets?

Reply:
Hmm, the hash function only returns once it is done with all assets in the album. Maybe it would be better if we convert it to the stream for better user experience